include/supl_os_client.rst

```diff
@@ -0,0 +1,35 @@
+.. _supl_client:
+
+SUPL client
+###########
+
+The Secure User-Plane Location (SUPL) client module integrates the external SUPL client library into the |NCS|.
+This library can be used to receive A-GPS data over the SUPL protocol.
+
+The SUPL client library is released under a different license than the |NCS|.
+Therefore, you must download and install it separately to be able to use this module.
+
+Downloading and installing
+**************************
+
+You can download the SUPL client library from Nordic Semiconductor's website.
```

Comment:
@bjorn-tore-taraldsen Where can it be downloaded?

Reply:
Still work in progress.
Will most likely be here https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF9160-DK/Download#infotabs 
Together with the other downloads on the bottom of the page.
Suggested header for this download section: "nRF9160 SiP SUPL Client library"